fontir/src/glyph.rs

```diff
@@ -435,6 +471,11 @@ impl Work<Context, WorkId, WorkError> for GlyphOrderWork {
         } else {
             trace!("No new glyphs, final glyph order == preliminary glyph order");
         }
+
+        let Some(0) = new_glyph_order.glyph_id(&NOTDEF) else {
+            todo!(".notdef must be present and must be gid 0")
```

Comment:
is this intentional? If we want to leave in a panic I'd write this as an `assert` instead.

Reply:
oh lol that's now guaranteed; this is leftover exporation that I will delete